tools/ssl-create.sh

```diff
@@ -85,12 +85,21 @@ if [ ! -r "${PEM_FINAL_CERT}" ] || [ ! -s "${PEM_FINAL_CERT}" ] || \
         exit $?
     fi
 
+    # certificate configuration (disable CA)
+    echo "[ req ]" > /run/tntnet-bios/ssl.conf
```

Comment:
I think the original script draft was with a `cat << EOF ...` construct
That one avoids extra I/Os at least, though this is a smaller concern on tmpfs.